NEXUS-26961: Updating documentation. (#15)
sonatype-nexus-community/nexus-blobstore-azure-cloud: README.md

    Eclipse Foundation. All other trademarks are the property of their respective owners.

-->

> This repository is no longer maintained as the Azure blob storage implementation has been moved into the Repository Manager codebase as a supported feature for pro users. Upgrade to the latest version of Repository Manager to use Azure backed blob stores (Pro feature). Also, issues for this project have been disabled. To open new issues relating to Azure blob stores create a new issue in JIRA.

Reviewer: ```suggestion
This project is no longer maintained as the Azure blob storage implementation has been moved into the Repository Manager codebase as a supported feature for Pro users. Upgrade to the latest version of Repository Manager to use Azure backed blob stores (Pro feature). Also, issues for this project have been disabled. To open new issues relating to Azure blob stores create a new issue in JIRA.
```
Minor wording.  I also minorly feel "(Pro feature)" isn't necessary as you said it the line before, but didn't feel strongly enough to include that in my github suggestion.

Reviewer: Also `To open new issues relating to Azure blob stores create a new issue in JIRA.` might be confusing, they might open issues related to this plugin there.  Consider striking, hopefully people know to file issues on our products in JIRA.

Nexus Repository Azure Blob Storage Blobstore
==============================

> This project adds [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/) backed blob stores to Sonatype Nexus Repository 3.16 and later. It allows Nexus Repository to store the components and assets in Azure Blobs instead of a local filesystem.

Reviewer: ```suggestion
This project adds [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/) backed blob stores to Sonatype Nexus Repository 3.16 thru 3.29.2. It allows Nexus Repository to store the components and assets in Azure Blobs instead of a local filesystem.
```
Rationale: "Cutoff date"?

`Dnxrm.azure.accountKey`. Integration tests will create temporary storage containers and tests should cleanup when 
complete. If you're running tests often check your storage account because it is likely that some containers may be left
behind and not properly cleanup up by the tests. 
 
     mvn clean install -P it -Dnxrm.azure.accountName=<accountName> -Dnxrm.azure.accountKey=<accountKey>

Installation
------------

> In Nexus Repository Manager 3.30.0+ Azure Blob Store support is available for pro users.

Reviewer: ```suggestion
In Nexus Repository Manager 3.30.0+ Azure Blob Store support is available for Pro users.
```
Rationale: Trivial caps (I think it's NXRM Professional, Pro fo short).

This plugin is not going to be updated or supported anymore. Use at your own risk.

> While nexus is stopped, copy target/nexus-blobstore-azure-cloud-0.5.0-SNAPSHOT-shaded.jar in the deploy folder in your

Reviewer: ```suggestion
While NXRM is stopped, copy target/nexus-blobstore-azure-cloud-0.5.0-SNAPSHOT-shaded.jar in the deploy folder in your
```
Final boyscout, can't help myself ;p

* Use this contribution at the risk tolerance that you have. 
* There are some incomplete features and known issues.

> * Do NOT file Sonatype support tickets related to Azure blob store support

Reviewer: ```suggestion
* Do NOT file Sonatype support or JIRA tickets related to this Azure blob store plugin
```
